app/views/sa/knownFacts.scala.html

```diff
@@ -42,6 +42,7 @@
         @components.input_text_field(
```

Comment:
This needs to change to use `@components.input_text` and remove the heading element, so that the h1 is the label, rather than having a h1 and a label that have the same text.

Reply:
And while we are at it we should do the same for the postcode page